setup.py

```diff
@@ -32,12 +32,20 @@
     "elasticsearch>=8.0.0,<9.0.0",
 ]
 
+async_requires = [
+    "elasticsearch[async]>=8.0.0,<9.0.0",
+]
+
 develop_requires = [
+    "elasticsearch[async]",
+    "unasync",
     "pytest",
     "pytest-cov",
     "pytest-mock",
+    "pytest-asyncio",
     "pytz",
     "coverage",
+    "mock",  # needed to have AsyncMock in Python <= 3.7
```

Comment:
No longer needed as we plan to drop Python 3.7 support.

Reply:
Removed